Makefile

```diff
@@ -327,7 +327,14 @@ lint: check-makefiles
 	# Note that we don't automatically suggest replacing sort.Float64s() with slices.Sort() as the documentation for slices.Sort()
 	# at the time of writing warns that slices.Sort() may not correctly handle NaN values.
 	faillint -paths \
-		"sort.{Strings,Ints}=golang.org/x/exp/slices.Sort" \
+		"sort.{Strings,Ints}=golang.org/x/exp/slices.{Sort}" \
```

Comment:
Note to reviewers: it wasn't working as expected. Functions must be wrapped by `{}`.